src/controllers/userpasswordpage.js

```diff
@@ -13,16 +13,14 @@ define(["loading", "libraryMenu", "emby-button"], function (loading, libraryMenu
                 if ("Guest" == user.ConnectLinkType) {
                     page.querySelector(".localAccessSection").classList.add("hide");
                     showPasswordSection = false;
-                } else {
-                    if (user.HasConfiguredPassword) {
+                } else if (user.HasConfiguredPassword) {
                         page.querySelector("#btnResetPassword").classList.remove("hide");
                         page.querySelector("#fldCurrentPassword").classList.remove("hide");
                         showLocalAccessSection = true;
                     } else {
                         page.querySelector("#btnResetPassword").classList.add("hide");
                         page.querySelector("#fldCurrentPassword").classList.add("hide");
                     }
```

Comment:
this should be de-indented to the left

Reply:
https://github.com/jellyfin/jellyfin-web/pull/509/commits/47cc029c7056b2ad334a93ba80f1568c6020a0ad